revocation/internal/crl/crl.go

```diff
@@ -279,9 +279,20 @@ func checkRevocation(cert *x509.Certificate, b *crl.Bundle, signingTime time.Tim
 		return nil, errors.New("baseCRL cannot be nil")
 	}
 
-	revocationListBundle := [][]x509.RevocationListEntry{b.BaseCRL.RevokedCertificateEntries}
-	if b.DeltaCRL != nil {
-		revocationListBundle = append(revocationListBundle, b.DeltaCRL.RevokedCertificateEntries)
+	// merge the base and delta CRLs in a single iterator
+	revocationListIter := func(yield func(x509.RevocationListEntry) bool) {
```

Comment:
Will it be better to do
```suggestion
	revocationListIter := func(yield func(*x509.RevocationListEntry) bool) {
```